sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -534,7 +536,9 @@ def get_subscription_receiver(
         :keyword Optional[float] max_wait_time: The timeout in seconds between received messages after which the
          receiver will automatically stop receiving. The default value is None, meaning no timeout. If connection
          errors are occurring due to write timing out, the connection timeout value may need to be adjusted. See
-         the `socket_timeout` optional parameter for more details.
+         the `socket_timeout` optional parameter for more details. If there is session enabled queue or topic and 
+         NEXT_AVAILABLE_SESSION is specified, max_wait_time will wait for that time to receive a message from any
```

Comment:
wording seems very slightly strange. Maybe something like this?
```suggestion
         NEXT_AVAILABLE_SESSION is specified, operation will time out after waiting `max_wait_time` seconds to receive a message from any session.
```